pkg/bot/slack_socket.go

```diff
@@ -216,6 +205,7 @@ func (b *SocketSlack) Start(ctx context.Context) error {
 						UserName:        userName,
 						CommandOrigin:   cmdOrigin,
 						State:           state,
+						EventTimeStamp:  callback.Message.Timestamp, // FIXME
```

Comment:
do we need to fix this in this PR?

Reply:
the comment is  already outdated, I will remove it 👍